caffe2/operators/channel_shuffle_op.cc

```diff
@@ -37,7 +37,8 @@ REGISTER_GRADIENT(ChannelShuffle, GetChannelShuffleGradient);
 OPERATOR_SCHEMA(ChannelShuffle)
     .IdenticalTypeAndShape()
     .NumInputs(1)
-    .NumOutputs(1);
+    .NumOutputs(1)
+    .InheritOnnxSchema("ChannelShuffle");
```

Comment:
Does it create a non-optional dependency on ONNX schema in Caffe2 schema?

Reply:
It doesn't. It's just the string property. Caffe2 can still be built without ONNX.